modules/apps/jaggery/mdm/src/mdm/modules/device.js

```diff
@@ -51,7 +52,7 @@ deviceModule = function () {
             log.error("User object was not found in the session");
             throw constants.ERRORS.USER_NOT_FOUND;
         }
```

Comment:
Why there are error logs in finally blocks? and it seems like there is something wrong with the logic here.